Application.groovy

```diff
@@ -1,7 +1,6 @@
-@Grab("edu.wisc.my.restproxy:json-proxy-service:1.0")
+@Grab("edu.wisc.my.restproxy:json-proxy-service:1.1.6-SNAPSHOT")
```

Comment:
Might want to change this to a release, I don't believe -SNAPSHOTs are deployed anywhere (and if they were, you'd need to indicate the repository in which the -SNAPSHOT would be found).